src/meltano/cli/upgrade.py

```diff
@@ -34,9 +34,16 @@ def upgrade(ctx, project):
 
     \b\nRead more at https://docs.meltano.com/reference/command-line-interface#upgrade
     """
-    engine, _ = project_engine(project)
-    upgrade_service = UpgradeService(engine, project)
-    ctx.obj["upgrade_service"] = upgrade_service
+    raise Exception(
```

Comment:
Would a more specific exception make sense here?

```suggestion
    raise CliError(
```